src/Loggable/Mapping/Driver/Xml.php

```diff
@@ -68,7 +68,7 @@ public function readExtendedMetadata($meta, array &$config)
         }
 
         if (!$meta->isMappedSuperclass && $config) {
-            if (is_array($meta->getIdentifier()) && count($meta->getIdentifier()) > 1) {
+            if ($meta instanceof \Doctrine\ODM\MongoDB\Mapping\ClassMetadata && is_array($meta->getIdentifier()) && count($meta->getIdentifier()) > 1) {
```

Comment:
Please, import the classes and apply the same change with all the new class usages introduced in this PR.